### PR TITLE
Support Laravel 8

### PR DESCRIPTION
**Pull request by Kristories**

### CODE REVIEW

File: .github/workflows/default.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
         dependency-version: [prefer-lowest, prefer-stable]
         os: [ubuntu-latest, windows-latest]
         include:
+          - laravel: 8.*
+            testbench: 6.*
```
Comment (melihovv) [inline .github/workflows/default.yml:20]: Can you also add this please?
```
        exclude:
            - laravel: 8.*
              php: 7.2
```

File: skeleton/.github/workflows/default.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ jobs:
         dependency-version: [prefer-lowest, prefer-stable]
         os: [ubuntu-latest, windows-latest]
         include:
+          - laravel: 8.*
+            testbench: 6.*
```
Comment (melihovv) [inline skeleton/.github/workflows/default.yml:20]: The same is here

File: .github/workflows/default.yml (hunk 1 above)
Comment (Kristories) [reply .github/workflows/default.yml:20]: Sure

File: skeleton/.github/workflows/default.yml (hunk 2 above)
Comment (Kristories) [reply skeleton/.github/workflows/default.yml:20]: Done

**Comment by melihovv** [comment]
Thank you, @Kristories 
I will make the new release in a couple of minutes

**Pull request merged**